src/pages/Internship.jsx

```diff
@@ -1,180 +1,152 @@
-{/* w-1440 3/4 */}
-{/*  */}
-
+import Navbar from "../components/Navbar"
 import React from "react";
 import Footer from "../components/Footer";
-import Instagram from '../images/Instagram.png'; 
-import fam_placeholder from '../images/fam_placeholder.png'
-import TotoPic from '../images/TotoPic.png'
-
+import HeaderBG from "../assets/InternHeaderBG.jpg"
+import Gear from "../assets/FullGear.png"
+import SunGear from "../assets/SunGear.png"
```

Comment:
check if you can replace these as svg, the reason for that is because you seen the slight pixelation of the images on my end